azure-pipelines.yml

```diff
@@ -250,31 +230,42 @@ jobs:
         - "/hw/top_englishbreakfast/Vchip_englishbreakfast_verilator"
 
 - job: execute_verilated_tests
-  displayName: Execute tests on the Verilated system (excl. slow tests)
+  displayName: Build and run tests on sim_verilator
   pool: ci-public
-  timeoutInMinutes: 120
-  dependsOn:
-    - chip_earlgrey_verilator
-    - sw_build
+  timeoutInMinutes: 180
+  dependsOn: lint
   steps:
   - template: ci/install-package-dependencies.yml
-  - template: ci/download-artifacts-template.yml
-    parameters:
-      downloadPartialBuildBinFrom:
-        - chip_earlgrey_verilator
-        - sw_build
+  - task: DownloadSecureFile@1
+    condition: eq(variables['Build.SourceBranchName'], 'master')
+    name: bazelCacheGcpKey
+    inputs:
+      secureFile: "bazel_cache_gcp_key.json"
+  - bash: echo "##vso[task.setvariable variable=bazelCacheGcpKeyPath]$(bazelCacheGcpKey.secureFilePath)"
+    condition: eq(variables['Build.SourceBranchName'], 'master')
+    displayName: Set the remote cache GCP key path
+  - bash: |
+      export GCP_BAZEL_CACHE_KEY=$(bazelCacheGcpKeyPath)
+      ci/bazelisk.sh test \
+        --build_tests_only=true \
+        --test_timeout=600,600,-1,-1 \
+        --local_test_jobs=4 \
+        --test_timeout_filters=short,moderate \
+        --test_tag_filters=verilator,-failing_verilator \
+        //sw/device/...
+        # TODO: include long tests after testing in CI
+        # --test_timeout_filters=short,moderate,long \
+    displayName: Build and execute tests
   - bash: |
-      # Install an additional pytest dependency for result upload.
-      pip3 install pytest-azurepipelines
-
       . util/build_consts.sh
-      pytest --version
-      pytest test/systemtest/earlgrey/test_sim_verilator.py \
-        -m "not slow" \
-        --log-cli-level=DEBUG \
-        --test-run-title="Run system tests with Verilator simulation (excl. slow tests)" \
-        --napoleon-docstrings
```

Comment:
Please could you remove the Verilator backend from the pytest framework as well? I'd rather not keep the dead code around because it's only going to confuse someone (probably me!)

Reply:
Done